.github/workflows/test.yml

```
@@ -350,8 +353,8 @@ jobs:
           push: false
           tags: maturin
           load: true
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
+          cache-from: type=registry,ref=maturin-buildcache
+          cache-to: type=registry,ref=maturin-buildcache,mode=max
```

> Test Docker failed for:
> 
> ```shell
> ERROR: failed to solve: error writing layer blob: server message: insufficient_scope: authorization failed
> Error: buildx failed with: ERROR: failed to solve: error writing layer blob: server message: insufficient_scope: authorization failed
> ```

I think it needs to include the registry hostname? Otherwise it assumes Docker Hub.

```suggestion
          cache-from: type=registry,ref=ghcr.io/${{ github.repository }}:maturin-buildcache
          cache-to: type=registry,ref=ghcr.io/${{ github.repository }}:maturin-buildcache,mode=max
```